Cargo.toml

```diff
@@ -102,5 +115,5 @@ tempfile = "3.8"
 
 # TODO: Keep around until alloy-contract is stable.
 # This should only be used in `alloy-contract` tests.
-# [patch.crates-io]
-# alloy-sol-macro = { git = "https://github.com/alloy-rs/core", rev = "18b0509950c90d9ec38f25913b692ae4cdd6f227" }
+[patch.crates-io]
+alloy-sol-macro = { git = "https://github.com/alloy-rs/core", rev = "ab0cab1047a088be6cffa4e7a2fcde7cf77aa460" }
```

Comment:
This is now on main so can remove I think? Or still needed

Reply:
will keep it until release